src/web/src/pages/signup.tsx

```diff
@@ -249,59 +253,42 @@ const SignUpPage = () => {
         return <RSSFeeds />;
 
       case SIGN_UP_STEPS.REVIEW:
-        return <Review />;
+        return <Review accountError={telescopeAccount.error} />;
```

Comment:
The current behavior redirects users to an error page if there is a problem creating a new account. It'll be much better if we show this error right in the form so they can edit the information and submit it again.